components/src/core/empty-state/empty-state.component.scss

```diff
@@ -30,3 +30,7 @@
         }
     }
 }
+
+h2 {
```

Comment:
You shouldn't need this rule anymore, there's class for pxb-empty-state-title that has margin-bottom: 0 on it.

Reply:
Avoid styling directly on elements.   Our components aren't using style encapsulation anymore so these styles will bleed into other components.  If we stick to using classes this shouldn't be an issue.